packages/styles/scss/components/data-table/_mixins.scss

```diff
@@ -0,0 +1,26 @@
+// @access public
```

Comment:
Might need a copyright banner at the top of this file

Reply:
Fixed 👍